### PR TITLE
Fixes #743: Fix 'title changing to since:day' at homepage

### LINKED ISSUE #743: <title> changes to since:day after trending hashtags are loaded at homepage
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
When the page is initially loaded, The title is `Loklak Search - Distributed....`, but when trending Hashtags are loaded, it changes to `since:day`.

**Expected behavior:**
Since we are on the home page, the title shouldn't change to `since:day`.

**Screenshots of the issue:**
screenshot from 2018-06-08 12-25-10

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Changes made to fix ```Title changing to since:day``` at homepage.

**Screenshots (if appropriate)** 
screenshot from 2018-06-09 20-40-41

**Link to live demo: http://pr-746-fossasia-loklaksearch.surge.sh** 

**Closes #743**

### CODE REVIEW

**Comment by simsausaurabh** [comment]
@singhpratyush @praveenojha33 Please review.

**Comment by praveenojha33** [comment]
Looks Good but currently but currently I am not getting results and hashtags( I think loklak server is down )

**Comment by simsausaurabh** [comment]
@praveenojha33 This is the server side problem which is not giving results currently. Please see http://loklak.org which is also not returning hastags.

**Comment by simsausaurabh** [comment]
@praveenojha33 please review it, now server is providing results.

**Comment by mariobehling** [comment]
This should not be committed to master directly. First commit to development and make a weekly merge from dev to master.

File: src/app/effects/title.effects.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import * as titleAction from '../actions/title';
+
+@Injectable()
+export class SetTitleEffects {
+
+	@Effect({ dispatch: false })
+	resetTitle$: Observable<void>
+		= this.actions$
+					.ofType(titleAction.ActionTypes.SET_TITLE)
+					.withLatestFrom(this.store$)
```
Comment (hemantjadon) [inline src/app/effects/title.effects.ts]: Remove this there is no need of store

File: src/app/actions/title.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * payload. Expressing actions as classes enables powerful
+ * type checking in reducer functions.
+ *
+ * See Discriminated Unions: https://www.typescriptlang.org/docs/handbook/advanced-types.html#discriminated-unions
+ */
+
+export class SetTitleAction implements Action {
+	type = ActionTypes.SET_TITLE;
+
+	constructor (public payload?: string) { }
```
Comment (hemantjadon) [inline src/app/actions/title.ts]: Payload should not be optional. It should be mandatory

**Comment by simsausaurabh** [comment]
@praveenojha33 The implemented solution was not correct. Since the project uses ngrx redux, it could be made more structured and properly using it.

**Pull request closed without merging**